Rocket.Chat/Extensions/UINavigationBar/UINavigationItem.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  UINavigationItem.swift
+//  Rocket.Chat
+//
+//  Created by Rudrank Riyam on 29/03/19.
+//  Copyright © 2019 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+extension UINavigationItem {
+    func moreButton() {
+        rightBarButtonItem?.accessibilityLabel = localized("auth.more")
```

Comment:
We have a separate localization method for accessibility:


https://github.com/RocketChat/Rocket.Chat.iOS/blob/develop/Rocket.Chat/Helpers/VoiceOver/VOLocalized.swift

So this should be like 
```swift
VOLocalizedString("auth.more")
```

Reply:
My apologies. I missed out on that. I have made the changes.